### PR TITLE
[Wallet] Implement new import wallet flow designs

### LINKED ISSUE #753: Show error when redeeming empty invite code / backup phrase
### Expected Behavior

We should not allow invite codes or  backup keys with no money to be redeemed
Backup Key- Error: the account you are trying to import has no balance!
Invite- Error: this invite code has no balance! Try using a different code or requesting a new one.

### Current Behavior

Unhelpful error?

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Implement new import wallet flow designs including empty wallet warning
- Improve responsiveness of import wallet screen
- More robust error handling for import saga
- Add tests for import saga
- Cleanup redundant web3 zerosync mocks introduced here: https://github.com/celo-org/celo-monorepo/pull/1108

### Tested

Ran through import flow with valid wallet
Ran through import flow with invalid wallet
Ran through import flow with empty wallet

### Related issues

- Fixes #753 

### Backwards compatibility

Yes

wa-import-v3
wa-import-v3-error
wa-import-empty

### CODE REVIEW

**Review by ashishb — APPROVED** [review]
Approving the cleanup part where you removed all the duplicate mocks I introduced in #1108

File: packages/mobile/src/import/saga.test.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { mockAccount } from 'test/values'
+
+const mockPhraseValid =
+  'oil please secret math suffer mesh retreat prosper quit traffic special creek educate rate weasel wide swing crystal day swim frost oxygen course expire'
+const mockPhraseInvalid =
+  'oilsome pleasely secretment math suffer mesh retreat prosper quit traffic special creek educate rate weasel wide swing crystal day swim frost oxygen course expire'
+
+describe('Import wallet saga', () => {
+  it('imports a valid phrase', async () => {
+    // @ts-ignore
```
Comment (jeanregisser) [inline packages/mobile/src/import/saga.test.ts:29]: What is it ignoring?

**Review by jeanregisser — APPROVED** [review]
Nice work! 👍

File: packages/mobile/src/import/saga.test.ts (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/import/saga.test.ts:29]: It seems like a bug with expectSaga's typings for saga parameters

**Pull request merged**